fetch_and_process_api_call/as/log.go

```diff
@@ -0,0 +1,29 @@
+package as
```

Comment:
I only did a quick check, but it looks like you are not using logging in the example and so this could be deleted (or you could add some logging in to the example, which could be a nice touch)

Reply:
reported a bug https://blocky.atlassian.net/browse/BKY-6321 and removed the file